build-system/pr-check/dist.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview
+ * This script builds the AMP runtime for production.
+ * This is run during the CI stage = build; job = dist.
+ */
+
+const colors = require('ansi-colors');
+const {
+  printChangeSummary,
+  startTimer,
+  stopTimer,
+  timedExecOrDie: timedExecOrDieBase,
+  uploadDistOutput} = require('./utils');
+const {isTravisPullRequestBuild} = require('../travis');
+const FILENAME = 'dist.js';
+const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
+const timedExecOrDie =
+  (cmd, unusedFileName) => timedExecOrDieBase(cmd, FILENAME);
+
+function main() {
+  const startTime = startTimer(FILENAME, FILENAME);
+
+  if (!isTravisPullRequestBuild()) {
+    timedExecOrDie('gulp dist --fortesting');
```

Comment:
Late realization: It's necessary to run `gulp update-packages` before this line in order to avoid errors like this: https://travis-ci.org/ampproject/amphtml/jobs/496622483#L634-L643

Reply:
Good catch. I noticed it in the Local Tests job too. Follow up PR - https://github.com/ampproject/amphtml/pull/20991